### PR TITLE
DOCSP-27625 Add Table Filter Icon Mention to Table Filters

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## Description

Adding mention of the filter icon which is shown when you add Table filters to tables in Relational Migrator.

## Staging

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-27625/table-filters/apply-table-filters/

## Jira

https://jira.mongodb.org/browse/DOCSP-27625

## Build

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64c916cb87b3b36046e84fde

### CODE REVIEW

File: source/table-filters/apply-table-filters.txt [hunk 1]
```diff
@@ -45,6 +45,9 @@ Steps
    radio button and enter a value. This applies a limit clause at the 
    end of your SQL query.
 #. Click :guilabel:`Save and close`.
+   Tables that have applied table filters display a :icon-fa5:`filter`
```
Comment (ajhuh-mdb) [inline source/table-filters/apply-table-filters.txt:48]: [suggestion] I think we can slightly reword this to avoid the confusion with the [participle](https://www.mongodb.com/docs/meta/style-guide/writing/clarify-gerunds-participles/#clarify-gerunds-and-participles), adopt more of an [active voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/#use-active-voice), and flow better from the preceding sentence. 

Maybe something along the lines of "After you save your filter, the :guilabel:`Schema model` pane displays a :icon-fa5:`filter` icon under the  :guilabel:`Relational` header."

**Review by ajhuh-mdb — CHANGES_REQUESTED** [review]
Thanks for making this update @ianf-mongodb ! Just one comment to help incorporate more active voice.

**Review by ianf-mongodb — COMMENTED** [review]
Thanks for the suggestion @ajhuh-mdb I have updated accordingly.

**Comment by ianf-mongodb** [comment]
Hi @tomhollander another small update for your consideration, thanks!

**Review by ajhuh-mdb — APPROVED** [review]
LGTM! Thanks for making these changes.

File: source/table-filters/apply-table-filters.txt [hunk 2]
```diff
@@ -45,6 +45,9 @@ Steps
    radio button and enter a value. This applies a limit clause at the 
    end of your SQL query.
 #. Click :guilabel:`Save and close`.
+   After you save your filter, the :guilabel:`Schema model` pane 
+   displays a :icon-fa5:`filter` icon under the :guilabel:`Relational` 
+   header.
```
Comment (tomhollander) [inline source/table-filters/apply-table-filters.txt:50]: Can we add "for any tables containing a filter" or similar? It's currently a bit vague.

**Review by tomhollander — CHANGES_REQUESTED** [review]

File: source/table-filters/apply-table-filters.txt (hunk 2 above)
Comment (ianf-mongodb) [reply source/table-filters/apply-table-filters.txt]: Sure thing, this has been updated:

```
   After you save your filter, the Schema model pane 
   displays a filter icon for any tables containing a filter 
   under the Relational header.
 ```

**Review by ianf-mongodb — COMMENTED** [review]
Updated

**Review by tomhollander — APPROVED** [review]
Thanks for the quick/late update!
